Cargo.toml

```diff
@@ -11,5 +11,8 @@ readme = "README.md"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bevy = "0.3.0"
-lyon = "0.16.0"
+bevy = {version = "0.4.0", default-features = false, features=["render"]}
+lyon = "0.16.2"
```

Comment:
❓ Does this need version number need to be constrained so tightly to a patch version? Could it be loosened to be constrained to just the minor version number instead?

Reply:
Same question for the `bevy` version number above.